espnet2/layers/stft.py

```diff
@@ -1,11 +1,13 @@
 from typing import Optional
-from typing import Tuple
+from typing import Tuple, Union
 
 import torch
 from typeguard import check_argument_types
+from torch_complex.tensor import ComplexTensor
 
 from espnet.nets.pytorch_backend.nets_utils import make_pad_mask
 from espnet2.layers.inversible_interface import InversibleInterface
+import torchaudio
```

Comment:
Very good to import torchaudio.
Let's try to find a place to use torchaudio functions as much as possible!